app/uk/gov/hmrc/digitalservicestax/aa_data/package.scala

```diff
@@ -93,7 +93,7 @@ package object data extends SimpleJson {
 
   type IBAN = String @@ IBAN.Tag
   object IBAN extends RegexValidatedString(
-    """^[0-9]"{4,50}""", // TODO
+    """^[0-9]{8}$""", // TODO
```

Comment:
8 characters seems very short for an IBAN.

Reply:
@hmrclt The existing regex is failing. The generator is also failing. That could be a short fall in the ScalaCheck Regex generator, however in the backend this is the regex we use for IBAN.

Reply:
@hmrclt Fixed using separate library, IBAN is not driven by regex, but by a checksum algorithm.